### ISSUE TITLE
Minor documentation issues Nutrient model

### ISSUE THREAD

**Issue by KirstenVer** [APSIMInitiative/ApsimX#11230]
### What happened?

Minor fix to memo clarifying N2OFraction node under Nitrification under Nutrients.
- The current text reads "Currently, no N2O is assumed to be produced during nitrification processes." This is **incorrect**. The value of the N2OFraction = 1, because the loss (specified separately under Nloss = 0.002) is all N2O. I.e. no N2.

Minor addition for memo clarifying N2OFraction node under Denitrification under Nutrients
- suggest to add Verburg Pasley et al 2025 (see link below) so people have access to the equation

Minor fixes memo under clarifying Nitrificaton node
- This should read Nitrification …. rather than Denitrification.
- Link to Probert et al paper not working: https://www.sciencedirect.com/science/article/pii/S0308521X97000280
- It would also be useful for the users to add the Verburg and Pasley et al 2025 review paper here:
Verburg K, Pasley HR, et al. (2024) Review of APSIM’s soil nitrogen modelling capability for agricultural systems analyses. Agricultural Systems 224, 104213. https://doi.org/10.1016/j.agsy.2024.104213

Denitrification memo:
-Same reference link change for Probert and possible addition of Verburg Pasley et al 2025

Hydrolysis memo:
-This also comes from CERES and is described in Godwin, D.C., Jones, A.C., 1991. Nitrogen dynamics in soil-plant systems. In: Modeling Plant and Soil Systems. John Wiley & Sons, Ltd, pp. 287–321. https://doi.org/
10.2134/agronmonogr31.c13.
-Suggest adding Verburg Pasley et al 2025 as well

Nutrient memo
-fix link to Probert et al as per above

### What Operating System are you on?

Windows

**Comment by David-Spadaro** [comment]
Hi Kirsten,

Just to clarify your first dot point that read:

Minor fix to memo clarifying N2OFraction node under Nitrification under Nutrients.
•	The current text reads "Currently, no N2O is assumed to be produced during nitrification processes." This is incorrect. The value of the N2OFraction = 1, because the loss (specified separately under Nloss = 0.002) is all N2O. I.e. no N2. 

Does it just need to be changed from N2O to read N2? Did I understand that correctly?

**Comment by KirstenVer** [comment]
Hi David,

Thank you for following up. Yes, that would fix the error.

**Comment by David-Spadaro** [comment]
Great.
